c10/core/DispatchKeySet.cpp

```diff
@@ -19,6 +19,7 @@ constexpr DispatchKeySet backend_dispatch_keyset = autogradother_backends |
         DispatchKey::PrivateUse3,
         DispatchKey::MLC,
         DispatchKey::HPU,
+        DispatchKey::ORT,
```

Comment:
The fact that MSNPU was not here was a bug?

Reply:
Yes